src/entities/mats-monitor-hrly-value.entity.ts

```diff
@@ -81,29 +88,28 @@ export class MatsMonitorHrlyValue extends BaseEntity {
     () => HrlyOpData,
     o => o.matsMonitorHrlyValues,
   )
-  @JoinColumn({ name: 'hour_id' })matsDerivedHrlyValues
+  @JoinColumn({ name: 'hour_id' })
+  matsDerivedHrlyValues;
```

Comment:
I think this is a typo, you can confirm with @fanwari1 but I don't see a relation between mats monitor hrly value and derived hrly values. So 'matsDerivedHrlyValues' can probably be deleted, leavin the relation name at hrlyOpData